understanding/20/labels-or-instructions.html

```diff
@@ -24,47 +23,44 @@ <h2>Intent of Labels or Instructions</h2>
       
       <p>The intent of this Success Criterion is not to clutter the page with unnecessary information
          but to provide important cues and instructions that will benefit people with disabilities.
-         Too much information or instruction can be just as much of a hindrance as too little.
+         Too much information or instruction can be just as harmful as too little.
          The goal is to make certain that enough information is provided for the user to accomplish
          the task without undue confusion or navigation.
       </p>
-      
-      <div class="note">
-         
-         <p>When labels are provided for input objects, the input object's relationship to the
-            label (or to redundant text serving as the label) must be programmatically determinable
-            or available in text per 
-            <a href="info-and-relationships">1.3.1: Info and Relationships</a>.
-         </p>
          
-      </div>
-      
-      
+      <p>This Success Criterion does not require that labels or instructions be correctly marked up or
+         identified/associated with their respective controls - this aspect is covered separately by
+         <a href="info-and-relationships">1.3.1: Info and Relationships</a>. It is possible for content
+         to pass this Success Criterion (providing relevant labels and instructions) while failing
+         Success Criterion 1.3.1 (if the labels or instructions aren't correctly marked up/identified/associated).
+      </p>
+      <p>Further, this Success Criterion does not take into consideration whether or not alternative methods of
+         providing an accessible name or description for form controls and inputs has been used - this aspect is
+         covered separately by <a href="name-role-value">4.1.2: Name, Role and Value</a>. It is possible
+         for controls and inputs to have an appropriate accessible name or description (e.g. using <code>aria-label="..."</code>)
+         and therefore pass Success Criterion 4.1.2, but to still fail this Success Criterion (if the labels or instructions
+         aren't presented to all users, not just those using assistive technologies).
+      </p>
+      <p>While this Success Criterion <em>requires</em> the use of labels for controls and inputs, whether or not
```

Comment:
I'm not sure why requires is em'd here. I also think that it makes it sound more like the label element to me in this phrasing. I'd prefer "While this Success Criterion requires that controls and inputs have labels, whether or not"

Reply:
fixed